src/server.cpp

```diff
@@ -3146,26 +3119,21 @@ u32 Server::addParticleSpawnerAll(u16 amount, float spawntime,
 	return id;
 }
 
-void Server::deleteParticleSpawner(const char *playername, u32 id)
+void Server::deleteParticleSpawner(std::string playername, u32 id)
```

Comment:
Don't do this! This copies the entire string! Do this instead: `const std::string &playername`